src/ProjectTemplates/Web.Spa.ProjectTemplates/content/Angular-CSharp/.template.config/template.json

```diff
@@ -25,7 +25,8 @@
         ".template.config/**"
       ],
       "copyOnly": [
-        "wwwroot/**"
+        "wwwroot/**",
+        "ClientApp/package-lock.json"
```

Comment:
@DamianEdwards can you clarify why you did this?

Is it because of the template engine bug that appeared a few days ago?

I believe the package-lock.json needs to be processed by the template engine, since it contains stuff like Company1.Webapplication that needs to "potentially" match your project name?

I bring this up because @HaoK is investigating an issue, and we are trying to lock down on it.

Reply:
Yeah Hao reached out to me. I hit the issue while testing this and made this change at @pranavkm's suggestion.